src/app/pages/choose-account/components/accounts.tsx

```diff
@@ -108,10 +113,14 @@ const AddAccountAction = memo(() => {
 
   return (
     <Box mb="loose" px="base-tight" py="tight" onClick={onCreateAccount} {...bind}>
-      <Stack isInline alignItems="center">
-        <Box size="16px" as={FiPlusCircle} />
-        <Text color="currentColor">Generate new account</Text>
-      </Stack>
+      <HStack alignItems="center">
+        <FiPlusCircle size="16px" />
```

Comment:
Maybe we can just use the `PlusIcon` here?

Reply:
Great, thanks! I've been copying your icon updates and had meant to ask you this one. 

I'm not even sure where this button is, do you know? It says `Generate new account` but the button I know of is `Create account`

Reply:
I believe this is in the transaction request popup when signing into an app.

Reply:
OK thanks. I'll test there